index.js

```diff
@@ -108,14 +108,17 @@ Client.prototype._write = function (obj, enc, cb) {
 }
 
 Client.prototype.sendSpan = function (span, cb) {
+  truncate.span(span, this._opts)
```

Comment:
I'd prefer we do this in the write method, to enable full functionality even when using directly as a stream.

Reply:
Fixed